bin/conjur-cli.rb

```diff
@@ -209,4 +209,16 @@ def test_select
   end
 end
 
+desc "Wait for the Conjur server to be ready"
+command :wait_for_conjur do |c|
+  c.action do |global_options,options,args|
+    puts "Waiting for Conjur to be ready..."
+    while `curl -o /dev/null -s -w '%{http_code}' $CONJUR_APPLIANCE_URL` != "200"
```

Comment:
How is `CONJUR_APPLIANCE_URL` going to be set? We know that the service is going to be on localhost. I think we may need the port number as an argument (default: 80).

Reply:
Since this is a Ruby function, I would rather see this in Ruby rather than shelling out to `curl`. Example along the lines of https://docs.ruby-lang.org/en/2.0.0/Net/HTTP.html

```ruby
res = Net::HTTP.get_response(URI("http://localhost:#{port}"))
break if res.is_a?(Net::HTTPSuccess)
STDOUT.write(".")
sleep 1
```

Reply:
`conjurctl wait_for_conjur` seems a bit redundant. Maybe just `conjurctl wait`?

Also a timeout would be another good option to have. "wait forever" is a Jenkins killer.